src/test/regress/expected/gp_dist_wait_status_optimizer.out

```diff
@@ -0,0 +1,7 @@
+BEGIN;
```

Comment:
You may ignore adding of `*_optimizer` test file if outputs are the same.

Reply:
Done